config/manager/manager.yaml

```diff
@@ -87,7 +87,11 @@ spec:
           name: manager
           terminationMessagePolicy: FallbackToLogsOnError
           securityContext:
+            capabilities:
```

Comment:
from JIRA, adding only `readOnlyRootFilesystem` should be sufficient, no? why also add this capabilities?

Reply:
I think @mpryc is trying to enforce the _least privilege principle._

Reply:
Yes. Adding `readOnlyRootFilesystem` is enough to fix the issue, but removing `capabilities` is a bit more into the direction on having minimal privilege required without harm on the pod functionality.

Reply:
ok

Reply:
With the current settings our container runs as `restricted-v2`:

https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/authentication_and_authorization/managing-pod-security-policies#default-sccs_configuring-internal-oauth

```text
Note
The restricted-v2 SCC is the most restrictive of the SCCs that is included by default with the system. 
```